python/CCLE_data.py

```diff
@@ -38,29 +42,123 @@ def common_celllines(a,b):
 
```

Comment:
You need to finish the code with `return common` to actually output the variable common.